src/Components/Server/src/Circuits/RemoteNavigationManager.cs

```diff
@@ -18,6 +18,8 @@ internal sealed partial class RemoteNavigationManager : NavigationManager, IHost
     private IJSRuntime _jsRuntime;
     private bool? _navigationLockStateBeforeJsRuntimeAttached;
     private const string _enableThrowNavigationException = "Microsoft.AspNetCore.Components.Endpoints.NavigationManager.EnableThrowNavigationException";
+
+    [FeatureSwitchDefinition("Microsoft.AspNetCore.Components.Endpoints.NavigationManager.EnableThrowNavigationException")]
```

Comment:
Is this used to get the value replaced based on `runtimeConfig.json`?

Reply:
No, this is compile-time attribute that can be helpful in trimming or Roslyn analysis.